pkg/api/frontendlogging/source_maps.go

```diff
@@ -0,0 +1,165 @@
+package frontendlogging
+
+import (
+	"io/ioutil"
+	"net/http"
+	"net/url"
+	"os"
+	"path/filepath"
+	"strings"
+	"sync"
+
+	sourcemap "github.com/go-sourcemap/sourcemap"
+
+	"github.com/getsentry/sentry-go"
+	"github.com/grafana/grafana/pkg/plugins"
+	"github.com/grafana/grafana/pkg/setting"
+)
+
+type sourceMapLocation struct {
+	dir      string
+	path     string
+	pluginID string
+}
+
+type sourceMap struct {
+	consumer *sourcemap.Consumer
+	pluginID string
+}
+
+type ReadSourceMapFn func(dir string, path string) ([]byte, error)
+
+func ReadSourceMapFromFs(dir string, path string) ([]byte, error) {
```

Comment:
Nit, but initialisms should be in consistent case in Go.
```suggestion
func ReadSourceMapFromFS(dir string, path string) ([]byte, error) {
```

Reply:
goddamit! one day...
- [ ] fix